### ISSUE TITLE
Cloudfront 'use origin cache headers' terraform

### ISSUE THREAD

**Issue by nickjmv** [hashicorp/terraform-provider-aws#21272]
We are setting up a CloudFront Distribution and are using AWS Provider v3.60.0.

We are having some trouble configuring the 'use origin cache headers' setting.
I read in #1994 that omitting the 3 TTL parameters should switch the option but it doesn't.

Are we overlooking something here?

**Comment by justinretzolk** [comment]
Hey @nickjmv 👋 Thank you for taking the time to file this issue. So that we have all of the necessary information to look into this, can you update the issue description with the information from the [issue template](https://github.com/hashicorp/terraform-provider-aws/blob/main/.github/ISSUE_TEMPLATE/Bug_Report.md)?

**Comment by Siivers** [comment]
I had an issue with this as well, on AWS provider v3.41.0 omitting the parameters it appears to set them to 0.

But when I set them as the following

min_ttl = 0
default_ttl = 86400
max_ttl = 31536000

In the AWS console for me it appears to be set to "Use origin cache headers"

**Comment by mguare** [comment]
I have the same problem. In the AWS console, those values are set to 0 
```
Terraform v1.1.2
provider registry.terraform.io/hashicorp/aws v3.70.0
```

**Comment by AbirAbbas** [comment]
bump on this, having the same issue

**Comment by Srokap** [comment]
> I had an issue with this as well, on AWS provider v3.41.0 omitting the parameters it appears to set them to 0.

Your fix worked for me in provider v4.15.1

**Comment by josborne** [comment]
> I had an issue with this as well, on AWS provider v3.41.0 omitting the parameters it appears to set them to 0.
> 
> But when I set them as the following
> 
> min_ttl = 0 default_ttl = 86400 max_ttl = 31536000
> 
> In the AWS console for me it appears to be set to "Use origin cache headers"

Confirming that @Siivers solution here worked for me in AWS provider v4.39.0.
